ci/env.py

```diff
@@ -2,7 +2,7 @@
 
 import os
 
-LIBTELIO_ENV_MOOSE_RELEASE_TAG = "v2.0.0-libtelioApp"
```

Comment:
Why is telio version upgraded here, @dfetti is already working on this via dedicated task

Reply:
It's not. the new version is v3.0.0-libtelioApp

Reply:
Oh, I was wrong. :man_facepalming: Sorry